plugins/modules/ec2_snapshot.py

```diff
@@ -71,39 +71,33 @@
     required: false
     default: 0
     type: int
-
 author: "Will Thames (@willthames)"
 extends_documentation_fragment:
 - amazon.aws.aws
 - amazon.aws.ec2
-
 '''
 
 EXAMPLES = '''
 # Simple snapshot of volume using volume_id
 - amazon.aws.ec2_snapshot:
     volume_id: vol-abcdef12
     description: snapshot of /data from DB123 taken 2013/11/28 12:18:32
-
```

Comment:
Do we have any guidance around the whitespace in here? Personally I find it easier to scan-read with the whitespace between examples